base/uk.ac.stfc.isis.ibex.opis/check_opi_info.py

```diff
@@ -0,0 +1,42 @@
+import os
+import win32api
+import xml.etree.ElementTree as ET
+import unittest
+
+DEFAULT_ROOT_DIR = os.path.join(os.path.dirname(__file__), "resources")
+
+
+class TestOpiInfo(unittest.TestCase):
+    def check_path_valid(self, path_str):
```

Comment:
This method only ever returns either a list with one item, or an empty list. Perhaps a cleaner signature would be to return a `str` describing the error or `None` (in case of no error).

In any case, for a method with a non-trivial signature, it is good to put it in a docstring.

Reply:
convention for "private" methods - start with underscore:

```suggestion
    def _check_path_valid(self, path_str):
```